README.md

```diff
@@ -26,12 +26,25 @@ Or install it yourself as:
 
 ```bash
 dream help
+Commands:
+  dream deploy [TYPE] -T, --targets=one two three  # Deploys to specified targets
```

Comment:
Do we need a specific example of multiple targets? Just curious if there are any gotchas with multiples.

Reply:
For OpsWorks, there aren't really any gotchas.  It builds the local cookbook _once_ and deploys it to all unique destinations specified by the stacks.

For `chef-solo` deployments, it still builds the local cookbook _once_ and deploys to each target that's stale.  The only caveat is if different targets require different ssh keys.  You an only provide one key for all targets:

`dream deploy solo -T user@host1 user@host2 -i /path/to/key.pem`